hermes-console-vue/json-server/server.js

```diff
@@ -0,0 +1,27 @@
+const topics = require('./topics.json');
```

Comment:
Change this file to typescript file. No js allowed :D

Reply:
server.js was removed